runtime/ft_test.go

```diff
@@ -766,3 +768,263 @@ func BenchmarkRuntimeFungibleTokenTransfer(b *testing.B) {
 
 	utils.RequireValuesEqual(b, nil, mintAmountValue, sum)
 }
+
+// TODO:
```

Comment:
The goal is to uncomment and use this version of the ExampleToken and have the transaction still work